src/app/_pages/join/join.page.scss

```diff
@@ -0,0 +1,60 @@
+mat-grid-tile {
+    background: #0277bd;
+    border-radius: 5px;
+}
+
+button { 
+    position: absolute; 
+    bottom: 5%;
+    right: 5%;
+    border: 0px solid #d6d6d6;
+    width: 150px;
+    height: 50px;
+}
+
+mat-selection-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    column-gap: 1em;
+    padding-right: 1.5em;
+    grid-template-rows: 1fr 1fr 1fr 1fr;
+}
+
+mat-selection-list>* {
+    flex: 0 0 20%;
+}
+
+.mat-list-option {
+    background: #383838;
+    margin-top: 10px; 
+    margin-bottom: 10px; 
+    margin-right: 10px; 
+    margin-left: 10px; 
+    border-radius: 5px;
+    align-content: flex-start;
+    text-align: center;
+    height: 15vh;
```

Comment:
vh units are based on the viewport height, so be careful that this will always be enough space, even on small devices in landscape.